src/helpers/backupCID.ts

```diff
@@ -0,0 +1,28 @@
+import axios, { AxiosError } from 'axios'
+import env from './env'
+
+const IPFS_PINNING_URL = 'https://api.filebase.io/v1/ipfs/pins'
+
+export default async function backupCID(cid: string) {
+  try {
+    await axios.post(
+      IPFS_PINNING_URL,
+      {
+        cid,
+        name: cid,
+        origins: [
+          `/ip4/${env.IPFS_IP_ADDRESS}/tcp/4001/p2p/${env.IPFS_ID}`,
+          `/ip4/${env.IPFS_IP_ADDRESS}/udp/4001/quic/p2p/${env.IPFS_ID}`,
+        ],
+      },
+      {
+        headers: {
+          Authorization: `Bearer ${env.FILEBASE_API_TOKEN}`,
+          'Content-Type': 'application/json',
+        },
+      }
+    )
+  } catch (e) {
+    if (e instanceof AxiosError) console.log(e.message)
```

Comment:
Let's `console.error` all errors, we should probably use sentry with tags here. Backups are too important, we shouldn't miss anything

Reply:
I'll setup Sentry in a separate PR, but I switched to `console.error` in [72d6806](https://github.com/BigWhaleLabs/obss-ipfs-uploader/pull/16/commits/72d6806c588ca23a6f70f148429f09ad7f59790f).